### ISSUE TITLE
Future of Groovy Blogs

### ISSUE THREAD

**Issue by rpalcolea** [sbglasius/groovyblogs#32]
Hi @sbglasius 

What's the future for this repo? migrate to `Grails 3.x`? implement pending issues? would like to contribute if you think Groovy Blogs will be there on a long term.

**Comment by sbglasius** [comment]
Hi @rpalcolea 

GroovyBlogs will definitely be here in the long run. I have had various plans with it, but getting it running on Grails 3.x is a primary.

Another plan would be to migrate it to an entire new application using Grails 3.2 and the AngularJS profile.

Let me know, what you think.

**Comment by rpalcolea** [comment]
Yeah I was wondering about the upgrade. I would like to contribute to that.

I don't have experience with AngularJS but definitely can help on that if you want to... hopefully the learning curve won't be that hard.

I'll take a deeper look on the current code but yeah I'm interested in contribute back to the community and I think this is a good project for it 👍
